sdk/eventhub/event-hubs/samples/sendEvents.ts

```diff
@@ -42,18 +42,34 @@ async function main(): Promise<void> {
     //   { body: { "message": "Hello World 2" } },
     //   { body: { "message": "Hello World 3" } }
     // ];
+    console.log("Sending single event...");
+    const scientist = listOfScientists[0];
+    producer.send({ body: `${scientist.firstName} ${scientist.name}` });
+
+    console.log("Creating and sending batch events...");
     for (let index = 0; index < listOfScientists.length; index++) {
       const scientist = listOfScientists[index];
-      events.push({ body: `${scientist.firstName} ${scientist.name}` });
+      const isAdded = eventDatabatch.tryAdd({ body: `${scientist.firstName} ${scientist.name}` });
+      if (!isAdded) {
+        console.log("Unable to add all events to the batch");
+        break;
+      }
     }
+    await producer.send(eventDatabatch);
+
     console.log("Sending batch events...");
+    for (let index = 0; index < listOfScientists.length; index++) {
```

Comment:
The example for sending array of events should be above the one for batch. Then it will follow the order of type defined for `eventData` argument in `send()`

Reply:
Agreed!